packaging/nuget/package.ps1

```diff
@@ -44,4 +44,4 @@ foreach ($ProjectName in $Projects) {
     }
 }
 
-Get-ChildItem $IntermediatePackagesDir -Filter *.nupkg | Copy-Item -Destination $PackagesDir
+Get-ChildItem $IntermediatePackagesDir -Filter *.nupkg | ? {$_.Name -NotLike "*.symbols.nupkg"} | Copy-Item -Destination $PackagesDir
```

Comment:
Not sure how much I like the inline filter, it wouldn't be too obvious on first glance, but I wouldn't block on it.